<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QWidget" name="sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="signup_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
   </property>
   <property name="text">
    <string>가입</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>가입정보</string>
   </property>
   <widget class="QLineEdit" name="inputId">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_le">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="bussiness_number_le">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사업자번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사업주명</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="owner_name_le">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사업주 전화번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="owner_phone_le">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check_duplicate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>중복확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>영업점 정보</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>영업점명</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="store_name_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>영업점 주소</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="store_addr_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>영업점 전화번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="store_phone_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>소개문구</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="store_intro_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>오픈시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>마감시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>배달 최소 주문금액</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_order_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>배달 가능지역</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deliver_area_le">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>원산지 정보</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>영양 성분 정보</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>알레르기 정보</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="origin_info_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nutritional_info_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="allergy_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="open_time_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>118</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="close_time_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>118</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="CityCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="GuGunCombo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
